modules/ensemble/lib/screen_controller.dart

```diff
@@ -194,6 +194,16 @@ class ScreenController {
         } else if (action.options?['replaceCurrentScreen'] == true) {
           routeOption = RouteOption.replaceCurrentScreen;
         }
+
+        if (action.options?['closeToasts'] ?? true) {
+          ToastController().closeToast();
```

Comment:
@hemish11 we talked about it in the standup that the default should be true for both navigateScreen and navigateModalScreen

Reply:
Yes, it defaults to true. So the `??` true verifies that if the `action.options?['closeToasts']` is null and when we don't pass any it null then it will default to true.

Reply:
got it, thanks